src/catalog/catalog-shared.md

```diff
@@ -9,6 +9,10 @@ If the [Shared Catalog]({% link stores/b2b-features.md %}) feature is enabled in
 
 For the Default (General) public shared catalog, You must need to assign products to display catalog on the storefront. By default, it is empty and does not contain any products.
 
+{:.bs-callout-info}
+When the [Shared Catalog]({% link stores/b2b-features.md %}) B2B feature is enabled in the Configuration, each category permission, within the whole Catalog, will be set to `Deny` for all Customer Groups automatically.
+Additionally, each new category will have the same `Deny`category permissions by default to prevent showing this category on the Storefront website, before assignment to the Shared Catalog.
```

Comment:
```suggestion
{:.bs-callout-info}
When the [shared catalog]({% link stores/b2b-features.md %}) B2B feature is enabled in the configuration, each category permission for the catalog is set to `Deny` for all customer groups automatically. Additionally, when a new category is created, it has the `Deny` category permissions by default to prevent showing that category on the storefront site before assignment to the shared catalog.
```

Reply:
Suggestion applied